ebean-migration/src/main/java/io/ebean/migration/runner/LocalMigrationResources.java

```diff
@@ -114,74 +137,49 @@ private boolean readResourcesForPath(String path) {
     if (platform != null && loadedFrom(path, platform)) {
       return true;
     }
-    addResources(scanForBoth(path));
-    Collections.sort(versions);
+    addResources(scanForMigrations(path));
```

Comment:
I don't yet follow this.  Looks like it scans regardless of the index file finding the [sql] migrations to run.